taskcluster/scripts/toolchain/android-gradle-dependencies.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -ex
+
+function get_abs_path {
+    local file_path="$1"
+    echo "$( cd "$(dirname "$file_path")" >/dev/null 2>&1 ; pwd -P )"
+}
+
+CURRENT_DIR="$(get_abs_path $0)"
+PROJECT_DIR="$(get_abs_path $CURRENT_DIR/../../../..)"
+
+pushd $PROJECT_DIR
+
+# We build everything to be sure to fetch all dependencies
+./gradlew --no-daemon assemble bundle test lint
+
```

Comment:
Is caching gradle itself of value (like is done [here](https://searchfox.org/mozilla-central/source/taskcluster/scripts/misc/android-gradle-dependencies/after.sh#19-26))?